tests/unit/amo/components/Errors/TestAuthExpired.js

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import ErrorComponent from 'amo/components/Errors/ErrorComponent';
+import AuthExpired, {
+  AuthExpiredBase,
+} from 'amo/components/Errors/AuthExpired';
+import { createApiError } from 'amo/api';
+import { loadErrorPage } from 'amo/reducers/errorPage';
+import { logOutUser as logOutUserAction } from 'amo/reducers/users';
+import {
+  dispatchSignInActions,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+import { API_ERROR_SIGNATURE_EXPIRED } from 'amo/constants';
+
+describe(__filename, () => {
+  let store;
+
+  const render = ({ ...props } = {}) => {
```

Comment:
You could deal with `store` a bit differently. First off, because it's declared outside of the `render` function, it doesn't actually need to be passed in via `props`. You could simply reference `store` inside `render`. 

But I would consider changing that, and *not* declaring it outside of `render`. As it's the only prop that `render` uses, you could simply allow `render` to accept a `store` argument, and default it to `dispatchSignInActions().store`. Then tests that don't need to do anything special with `store` (like "renders a not authorized error") wouldn't even need to reference `store` at all.

Tests that do need to do something with `store` (like "logs out the user") could simply create `store` inside the test case and then pass it into `render` via the `store` argument. I feel like this is a bit cleaner, and it gets rid of both the `...props`  argument to `render` as well as the `beforeEach`. I think you're going to need to add at least one more test to the file, so I think it's worthwhile, but it's just a suggestion.